pyswmm/output.py

```diff
@@ -385,20 +385,25 @@ def _load_pollutants(self) -> NoReturn:
 
     @property
     @output_open_handler
-    def unit(self) -> int:
+    def units(self) -> int:
         """
-        Return SWMM output binary file unit type from swmm.toolkit.shared_enum.UnitSystem
+        Return SWMM output binary file units for unit system, flow, and pollutants
 
-        :returns: integer indicating unit system (0 = US, 1 = SI)
-        :rtype: int
+        :returns: list of integers indicating units for
+                  [unit system, flow, and pollutant]
+
```

Comment:
@jennwuu, sorry just saw this.. How about returning the string unit type instead of the index?

Reply:
I can break the units function into:
system_units,
flow_units,
pollutant_units.

Thoughts?

Reply:
Maybe look at what we've done in Simulation.py and return a dictionary?

Reply:
{'system': <UnitSystem.US: 0>, 'flow': <FlowUnits.CFS: 0>, 'pollutant': <ConcUnits.NONE: 3>}

Reply:
@jennwuu what is an example return? Are they strings or enumerations?

Reply:
They're enumeration from swmm-toolkit

Reply:
@jennwuu, I'm imagining the actual string like we have in `simulation.py`:

https://github.com/OpenWaterAnalytics/pyswmm/blob/6cf3d43fae07a48522401ab80fa18b883781611a/pyswmm/simulation.py#L561-L595

Reply:
Ok updated: {'system': 'US', 'flow': 'CFS', 'pollutant': 'NONE'}